resources/views/contacts.blade.php

```diff
@@ -84,12 +84,12 @@
                             <div class="staff__item_top">
                                 <h3 class="staff__item_bold text-20">{{ $item->{'position_'.app()->getLocale()} }}</h3>
                                 <h4 class="staff__item_bold text-18">
-                                    {!! $item->{'name_'.app()->getLocale()} ?? __('contact.full_name') !!}
+                                    {!! $item->{'name_'.app()->getLocale()} ?? ' ' !!}
```

Comment:
@hambardzumianvahag Is it important to have empty string here?
I think there is not make sense.
Here can be this

```
@if (!empty($item->{'name_'.app()->getLocale()}))
<h4 class="staff__item_bold text-18">
    {!! $item->{'name_'.app()->getLocale()} !!}
</h4>
@endif
```

Reply:
without empty string we have violation. and we discussed and thought that in this way was better